src/MatBlazor.Demo/Demo/DemoMatSlider.razor

```diff
@@ -283,3 +283,51 @@
     ")></BlazorFiddle>
     </SourceContent>
 </DemoContainer>
+
+<h5 class="mat-h5">Value update</h5>
+<DemoContainer>
+    <Content>
+        <MatTextField @bind-Value="@Value" Label="Value"></MatTextField>
+        <MatTextField @bind-Value="@ValueMin" Label="ValueMin"></MatTextField>
+        <MatTextField @bind-Value="@ValueMax" Label="ValueMax"></MatTextField>
+        <MatSlider TValue="decimal" Value="@Value" ValueMin="@ValueMin" ValueMax="@ValueMax" ValueChanged="@OnValueChanged2"></MatSlider>
```

Comment:
OnValueChanged2 is a pretty generic name that doesn't indicate what is changing. It may make sense to modify this so the dev knows _what_ value is changing when looking at the variable.

Reply:
i replied in this comment: https://github.com/SamProf/MatBlazor/pull/725/files#r501237630
i think it is the same case.
value is named **Value** so i think **OnValueChanged2** is pretty self explanatory, except the suffix of course, but that can be guessed from the fact that there are two methods in tha same file